### DISCUSSION TITLE
Delete Sql Scripts

### DISCUSSION THREAD

**Discussion by fatihtastemur** [dbeaver/dbeaver#11225] [Q&A] [answered]
How to delete a saved sql scripts in Dbeaver?

**Comment by serge-rider** [comment, marked as answer]
You can explore saved scripts in Project Explorer view->Scripts.
Just select script(s) and press <kbd>DEL</kbd>. 
Also you can select current script in explorer by clicking "Link with editor" button in Project Explorer view toolbar.

**Comment by tati-kru** [comment]
It looks like this question was solved. It will be closed. If you think that it is actual just let us know.

**Comment by ducciopiovani** [comment]
Hello I am working on a Mac and this didn't work for me. I have tried everything I could think of but still didn't succeed.

**Comment by ducciopiovani** [comment]
I solved it by right clicking on the scripts name once its opened and selecting delete script.

**Comment by manijang2** [comment]
@ducciopiovani  Thank you very much.. I am solved

**Comment by teneighty-yvr** [comment]
Manjaro Linux: The delete key did nothing in the project explorer. Right clicking on the tab of an open script did provide the option to delete. Thank you @ducciopiovani 

To manage scripts in bulk, I found the scripts stored in `~/.local/share/DBeaverData/workspace6/General/Scripts`.
